mlp/start_analysis.py

```diff
@@ -35,7 +37,8 @@ def main(sc):
     rdd = sc.textFile(user_rdds)
     models = rdd.map(lambda user_id:
                      learn_model_from_data(dataset_directory=os.path.join(root_directory, 'datasets', user_id),
```

Comment:
Ok yes the spark code does train one model per user. But it needs the scala code that reads the data out of cassandra, stores it into one directory per user and hands over the directories to python.